routing/turns_notification_manager.cpp

```diff
@@ -73,12 +73,13 @@ NotificationManager NotificationManager::CreateNotificationManagerForTesting(
 
 string NotificationManager::GenerateTurnText(uint32_t distanceUnits, uint8_t exitNum,
                                              bool useThenInsteadOfDistance, TurnItem const & turn,
-                                             measurement_utils::Units lengthUnits) const
+                                             measurement_utils::Units lengthUnits,
+                                             std::string const & nextStreet = "") const
```

Comment:
Hm, default value = "" is not needed in cpp function's definition. Only in declaration.